resources/wikia/modules/breakpointsLayout.js

```diff
@@ -0,0 +1,65 @@
+define('wikia.breakpointsLayout',  function() {
+	'use strict';
+
+	var WIDTH_OUTSIDE_MIN = 768,
+		CONTENT_BORDER = 1,
+		BREAKPOINT_LARGE_CONTENT_WIDTH = 1238,
+		BREAKPOINT_LARGE_PADDING_WIDTH = 30,
+		BREAKPOINT_MEDIUM_CONTENT_WIDTH = 1024,
+		BREAKPOINT_MEDIUM_PADDING_WIDTH = 20,
+		BREAKPOINT_MIN_CONTENT_WIDTH = 768,
+		BREAKPOINT_MIN_PADDING_WIDTH = 24,
+		RIGHT_RAIL_WIDTH_PLUS_SPACING = 320;
+
+	function getArticleContentMaxWidth(isWidePage) {
+		var articleWidth = BREAKPOINT_LARGE_CONTENT_WIDTH - 2 * BREAKPOINT_LARGE_PADDING_WIDTH;
+		return isWidePage ? articleWidth : articleWidth - RIGHT_RAIL_WIDTH_PLUS_SPACING;
+	}
+	function getArticleContentMediumWidth(isWidePage) {
+		var articleWidth = BREAKPOINT_MEDIUM_CONTENT_WIDTH - 2 * BREAKPOINT_MEDIUM_PADDING_WIDTH;
+		return isWidePage ? articleWidth : articleWidth - RIGHT_RAIL_WIDTH_PLUS_SPACING;
+	}
+
+	function getArticleContentMinWidth() {
+		return BREAKPOINT_MIN_CONTENT_WIDTH - 2 * (BREAKPOINT_MIN_PADDING_WIDTH + CONTENT_BORDER);
+	}
+
+	function getArticlePadding() {
+		return BREAKPOINT_MEDIUM_PADDING_WIDTH;
+	}
+
+	function getArticleWidth(breakpoint, isWidePage) {
+		var maxArticleWidth = getArticleContentMaxWidth(isWidePage),
```

Comment:
We don't need to calculate it before `switch`. Maybe it would be good idea to return function result directly instead of writing it to variable first?